image_tools_py/README

```diff
@@ -0,0 +1,109 @@
+This is a demonstration of the Quality of Service (QoS) features of ROS 2 using Python.
```

Comment:
@clalancette nit: can we make this a Markdown, add a top level header, use `literals` where appropriate and ```code blocks``` where appropriate?

Reply:
All good points.  I've cleaned up the whole README in 0dc878e7c1e38c37b3e885592eae84e610f66a9d (this applies to all of the README comments below).